a10_octavia/controller/worker/tasks/a10_network_tasks.py

```diff
@@ -835,6 +825,17 @@ def execute(self, vthunder, lb_resource, vrid_list, subnet,
         owner = vthunder.ip_address + "_" + vthunder.partition_name
         self._add_vrid_to_list(updated_vrid_list, subnet, owner)
         for vrid in updated_vrid_list:
+            vrid_subnet = self.network_driver.get_subnet(vrid.subnet_id)
+            if vrid_subnet.ip_version == 6:
```

Comment:
We do this IP_address, IP_address_cfg, ... string assignment couple times.
- maybe we can consider provide a function for this.
- consider put those string in a10constants.py

Reply:
DONE